src/server/scripts/Events/midsummer.cpp

```diff
@@ -218,7 +222,9 @@ class spell_midsummer_ribbon_pole : public AuraScript
             Creature* cr = target->FindNearestCreature(NPC_RIBBON_POLE_DEBUG_TARGET, 10.0f);
             if (!cr)
             {
-                target->RemoveAura(SPELL_RIBBON_POLE_CHANNEL_VISUAL);
+                target->RemoveAura(SPELL_TEST_RIBBON_POLE_CHANNEL_BLUE);
```

Comment:
You need to validate all spells used in scripts.

Reply:
```cpp
    bool Validate(SpellInfo const* /*spell*/) override
    {
        return ValidateSpellInfo({ SPELL_CRAB_DISGUISE });
    }
```
Example taken from earlier in the file

Reply:
Alright will update this